src/foam/nanos/auth/PermissionedPropertyDAO.js

```diff
@@ -141,7 +141,7 @@ foam.CLASS({
       ],
       javaCode: `
       String      of   = oldObj.getClass().getSimpleName().toLowerCase();
-      FObject     obj  = oldObj.fclone();
+      FObject     obj  = oldObj;
```

Comment:
do we even need this variable? Also out of curiosity, why the argument to this method is called "oldObj" but we actually passing the new object that is being put? And the "put" method also clones the object before calling this method which is also unnecessary  I believe

Reply:
This is the eventual return value. But oldObj is already cloned on line 261.

Reply:
Sorry, the comment about cloning was in regards to the put on HidePropertiesSink, so i guess invalid.
I dont understand the obj variable tho. Somewhere at the end of the method we are clearing axioms on the "obj" and then return it. With this change on line 144, clearing axioms on "obj" would be same as clearing axioms on "oldObj" so why dont we just clear them on oldObj directly and then return it? if there is any point of returning at all